app/controllers/api/class_members_controller.rb

```diff
@@ -33,8 +40,15 @@ def create
     end
 
     def create_batch
-      students = SchoolStudent::List.call(school: @school, token: current_user.token, student_ids: create_batch_params)
-      result = ClassMember::Create.call(school_class: @school_class, students: students[:school_students])
+      teacher_objects = create_batch_params.select { |user| user[:type] == 'teacher' }
+      student_objects = create_batch_params.select { |user| teacher_objects.pluck(:user_id).exclude?(user[:user_id]) }
```

Comment:
maybe I'm missing something and this has changed now you're not including owners, but can't you just select students by type?